gettingstarted.tex

```diff
@@ -45,13 +45,13 @@ \subsubsection{CTRL}
 
 Holding \specialkey{CTRL} and pressing \megakey{*} enters the Matrix Mode Debugger (refer to the {\bf MEGA65 Book} for more details).
 
-\subsubsection{RUN STOP}
+\subsubsection{RUN/STOP}
 \index{Keyboard!RUN STOP}
-Normally, pressing \specialkey{RUN\\STOP} stops the execution of a program. Holding \specialkey{SHIFT} while pressing \specialkey{RUN\\STOP} {\bf RUN}s the first program from disk.\index{BASIC 65 Commands!RUN}
+The \specialkey{RUN\\STOP} is the RUN / STOP key and is located at the top left-hand side of the keyboard. Normally, pressing \specialkey{RUN\\STOP} stops the execution of a program. Holding \specialkey{SHIFT} while pressing \specialkey{RUN\\STOP} {\bf RUN}s the first program from disk.\index{BASIC 65 Commands!RUN}
 
 Some programs override the \specialkey{RUN\\STOP} key and cannot be stopped in this way.
 
-You can boot your MEGA65 into the {\bf Machine Code Monitor}\index{Machine Code Monitor} by holding down \specialkey{RUN\\STOP} and pressing reset on the left-hand side of the computer.
+You can boot your MEGA65 into the {\bf Machine Code Monitor}\index{Machine Code Monitor} by holding down \specialkey{RUN\\STOP} and pressing reset (or powering up) on the left-hand side of the computer.
```

Comment:
"(or while switching on the computer)"

(I personally use something other than "switch on the computer" in other writing, but this was established as style for these docs prior to me, and I agree we should use a consistent phrase, at least in the UG.)

Reply:
Agree, see below comment.